### ISSUE TITLE
4Cker with 1 condition, no replicates

### ISSUE THREAD

**Issue by tlgolan** [rr1859/R.4Cker#6]
Hi,
I am trying to create '4Cker' object by using this code:
my_obj = createR4CkerObjectFromDFs(dfs =counts_W_PE,
                                   bait_chr="chr5",
                                   bait_coord= start_primer,
                                   bait_name = "Nppb",
                                   primary_enz = "AGATCT",
                                   samples ="PE_Nppb_1",
                                   conditions = "PE",
                                   replicates = 1,
                                   species = "rn",
                                   output_dir = getwd())
While counts_W_PE is a data frame object (created in R), start_primer- the start position of the primer, the species is Rattus Norvegicus. Unfortunately, I don't have biological replicates in this 4Cseq experiment.
For some reason I get this error: 

**Error in createR4CkerObjectFromDFs(dfs = counts_W_PE, bait_chr = "chr5",  : 
  Number of samples does not match**

I will be glad for some insights.
Thanks,
Tal

**Comment by rr1859** [comment]
Do you only have 1 column in counts_W_PE?

Ramya

**Comment by rr1859** [comment]
counts_W_PE should be a 4 column data.frame with chr, start,end, count for each fragment - is that what you have?

**Comment by tlgolan** [comment]
This is exactly what I have 

    On Thursday, July 21, 2016 10:40 PM, Ramya Raviram <notifications@github.com> wrote:
 

 counts_W_PE should be a 4 column data.frame with chr, start,end, count for each fragment - is that what you have?—

**Comment by tlgolan** [comment]
No, I have four columns: chr name, start position of the fragment, end position of the fragment and counts 

    On Thursday, July 21, 2016 10:36 PM, Ramya Raviram <notifications@github.com> wrote:
 

 Do you only have 1 column in counts_W_PE?

Ramya

**Comment by tlgolan** [comment]
For example:> head(counts_W_PE)    V1    V2    V3 Weizmann_PE1 chr5    43  3301           02 chr5  3308  3776           03 chr5  3783  6299           04 chr5  6306  8431           05 chr5  8438 11767           06 chr5 11774 12246           0 

    On Thursday, July 21, 2016 10:40 PM, Ramya Raviram <notifications@github.com> wrote:
 

 counts_W_PE should be a 4 column data.frame with chr, start,end, count for each fragment - is that what you have?—

**Comment by rr1859** [comment]
So for the function, dfs is just variable names for the different data frames so I would change it to
my_obj = createR4CkerObjectFromDFs(dfs =c("counts_W_PE"),
 bait_chr="chr5",...

**Comment by tlgolan** [comment]
Great! 
It works!
Thanks
